src/aks-preview/azext_aks_preview/_consts.py

```diff
@@ -136,7 +137,8 @@
     "confcom": CONST_CONFCOM_ADDON_NAME,
     'open-service-mesh': CONST_OPEN_SERVICE_MESH_ADDON_NAME,
     'azure-keyvault-secrets-provider': CONST_AZURE_KEYVAULT_SECRETS_PROVIDER_ADDON_NAME,
-    'gitops': CONST_GITOPS_ADDON_NAME
+    'gitops': CONST_GITOPS_ADDON_NAME,
+    'web_application_routing': CONST_WEB_APPLICATION_ROUTING_ADDON_NAME
```

Comment:
The key here will be what the user needs to enter when a certain addon should be enabled (like `--enable-addons monitoring` in command `aks create` to enable omsagent).

Since a separate model is used, the value corresponding to `web_application_routing` will not be used in addon_profiles.

Reply:
The value corresponding to web_application_routing has been removed. I have also added a code comment explaining why there's no value.